lerna.json

```diff
@@ -3,5 +3,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "2.4.1"
+  "version": "2.4.3"
```

Comment:
You don't need to change any version numbers as we handle them with our release process